README.md

```diff
@@ -60,12 +60,17 @@ export const walletLink = new WalletLink({
 })
 
 // Initialize a Web3 Provider object
-export const ethereum = walletLink.makeWeb3Provider(ETH_JSONRPC_URL, CHAIN_ID)
+export const ethereum = walletLink.makeWeb3Provider(DEFAULT_ETH_JSONRPC_URL, DEFAULT_CHAIN_ID)
 
 // Initialize a Web3 object
 export const web3 = new Web3(ethereum as any)
 ```
 
+Walletlink will use an rpcUrl provided by Coinbase Wallet clients regardless of the rpcUrl passed into `makeWeb3Provider` 
+for whitelisted networks. Walletlink solely needs an rpcUrl to be provided by the dapp as a fallback.
```

Comment:
Consider active rather than passive: 

```
< Walletlink solely needs an rpcUrl to be provided by the dapp as a fallback.
---
> Walletlink needs the dapp to provide an rpcUrl as a fallback.
```